spec/controllers/your_details_controller_spec.rb

```diff
@@ -70,34 +70,30 @@ module Wpcc
 
     describe '#create' do
```

Comment:
The context grouping here is all related to the new method. I could regroup it to be for session but I think the current grouping by method works. 

I struggled a bit with the best way to test the session expiring and I think the approach I've taken is ok, but I'm open to comments.